layouts/partials/compliance/insights.html

```diff
@@ -0,0 +1,39 @@
+<section class="mt-8 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>Pulumi Insights</h3>
+        <p class="py-4">
+            Use <a href="/product/pulumi-insights/">Pulumi Insights</a> to gain visibility into your cloud infrastructure's
+            configuration to assess {{ .Params.framework }} compliance. Pulumi Insights is Intelligence for Infrastructure as Code.
+            It adds advanced search, analytics, and AI to infrastructure as code.
+        </p>
+        <img src="https://www.pulumi.com/uploads/pulumi-insights-search.gif" />
+        <p class="py-4">
+            Use Copilot to assist configuring your infrastructure to make it compliance ready. You can tap into the Pulumi Copilot's
```

Comment:
I think there should be a H3 here that says Pulumi Copilot so its not just a run on from the Insights section